csm_web/scheduler/views.py

```diff
@@ -10,6 +10,9 @@
 from rest_framework import mixins
 from .serializers import CourseSerializer, SectionSerializer, StudentSerializer, AttendanceSerializer, MentorSerializer, OverrideSerializer
 from django.core.exceptions import ObjectDoesNotExist
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Something like "scheduler.views" would be better instead of __name__, especially if we have more apps in the future where we log from views

Reply:
This is dynamically populated with the value `scheduler.views`. `__name__` refers to the name of the current module in python.

Reply:
Ah, I thought it would be the name of the file

(also, didn't realize that __ meant bold in markdown)